README.md

```diff
@@ -597,6 +597,56 @@ These are tips to help you with a successful project workflow
    }
    ```
 
+1. You can use [`Create nexus types`](https://github.com/AhmedElywa/create-nexus-type) tool to generate `objectType` , `queryType` and `mutationType` from your `schema.prisma` file for every model.
+
+    **Example**
+    
+    ```prisma
+    model User {
+      id        String   @id @unique @default(cuid())
+      email     String   @unique
+      birthDate DateTime
+      posts     Post[]
+    }
+    ```
+    
+    Output
```

Comment:
Would help if you show the command / script user has to run.

Reply:
i will add full docs like in my Readme page if this help